spec/helpers/course/assessment/answer/programming_helper_spec.rb

```diff
@@ -17,7 +17,9 @@ def hello:
         end
       end
 
+      before { controller.define_singleton_method(:current_user) { nil } }
```

Comment:
so the helper can use can?. the helper doesn't have a controller so current_user returns a TypeError.